drivers/adc/adc_stm32.c

```diff
@@ -873,7 +873,7 @@ static void adc_stm32_setup_channels(const struct device *dev, uint8_t channel_i
 		__LL_ADC_CHANNEL_TO_DECIMAL_NB(LL_ADC_CHANNEL_VREFINT) == channel_id) {
 		adc_stm32_set_common_path(dev, LL_ADC_PATH_INTERNAL_VREFINT);
 	}
-#if defined(CONFIG_STM32_VBAT)
+#if defined(LL_ADC_CHANNEL_VBAT)
```

Comment:
Need to clean up the #endif below.

Reply:
Oops, good catch. I have just fixed that.